dart/lib/src/sentry_options.dart

```diff
@@ -289,6 +289,17 @@ class SentryOptions {
   /// to be sent to Sentry.
   TracesSamplerCallback? tracesSampler;
 
+  double? _profilesSampleRate;
```

Comment:
we have a [profilesSampler](https://github.com/getsentry/sentry-cocoa/blob/main/Sources/Sentry/Public/SentryOptions.h#L371) option in the other SDKs, which, if set, is prioritized over profilesSampleRate
I'm fine adding it in another PR, if we decide so